docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,1344 @@
+{
```

Comment:
<p>Ummm, I think you should demonstrate how to get to this as this is not the standard representation of Pearson correlation! Typically, it is the covariance divided by the standard deviations of each subsequence. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
I used line(14) and line(22) of the notebook [Matrix_Profile_Derivation](https://github.com/TDAmeritrade/stumpy/blob/main/docs/Matrix_Profile_Derivation.ipynb) to notice that the `q` in the paper is actually Pearson Correlation \rho.  I will add this note to the notebook. Does that suffice?

Reply:
<p>I think one should start with a standard definition of <code>\rho</code> (see wikipedia) and then manipulate it into the form below </p>

Reply:
Got it. Will do it.

Reply:
The formula I used in the notebook is actually eq(4) of this paper: [paper](https://www.cs.unm.edu/~mueen/Projects/JOCOR/joinICDM.pdf). I was wondering if I should better provide the reference rather than deriving it (which is 2 line at the most...but I think providing ref reduces the noise in the notebook and distracts readers less.)

What do you think?

Reply:
Sure. I think it is similar to software where we want to have minimum dependencies. I will add the calculation.